android/src/main/java/com/reactnativecommunity/webview/RNCWebViewMapManager.kt

```diff
@@ -0,0 +1,8 @@
+package com.reactnativecommunity.webview
+
+import android.webkit.WebView
+
+object RNCWebViewMapManager {
+  val rncWebViewMap = mutableMapOf<String, RNCWebView>()
+  val internalWebViewMap = mutableMapOf<String, WebView>()
```

Comment:
I've used the `type` WebView since Kotlin was complaining that the InternalWebView is a protected class and shouldn't be referenced here